first_use

```diff
@@ -141,137 +141,139 @@ finish_default_obfs4_step1()
 
 # This will completely remove pi (only for Raspbian OS). The parameter
 # will be set at the end of the run_install.sh
-if [ $FRESHINSTALLED = "FRESH_INSTALLED=1" ]; then
-	clear
- 	sudo sed -i "s/^FRESH_INSTALLED=.*/FRESH_INSTALLED=3/" ${RUNFILE}
-	if [ -d "/home/pi" ]; then
-	 	sudo killall systemd
-	 	sleep 5
-		sudo userdel -r pi
-	fi
-	clear
-	echo -e "${WHITE}[+] Fresh installation detected - we will reboot TorBox!${NOCOLOR}"
-	sleep 5
-	sudo reboot
+case $FRESHINSTALLED in
 
-# This will generate new unique SSH server keys (only for images)
-elif [ $FRESHINSTALLED = "FRESH_INSTALLED=2" ]; then
-	clear
-	sudo sed -i "s/^FRESH_INSTALLED=.*/FRESH_INSTALLED=3/" ${RUNFILE}
-	echo -e "${WHITE}[+] First time starting your TorBox!${NOCOLOR}"
-	echo -e "${WHITE}[+] Generating new unique SSH server keys for your TorBox installation!${NOCOLOR}"
-	(sudo rm -r /etc/ssh/ssh_host_*) 2> /dev/null
-	sudo ssh-keygen -A
-	echo ""
-	echo -e "${WHITE}[+] To take effect, you have to close and reopen all your SSH sessions.${NOCOLOR}"
-	FRESHINSTALLED="FRESH_INSTALLED=3"
```

Comment:
The meaning of `FRESHINSTALLED="FRESH_INSTALLED=3"` is that after running this code block, the next (`elif [ $FRESHINSTALLED = "FRESH_INSTALLED=3" ]; then`) will be executed.